libs/SmartStore/SmartStoreTests/SFSmartStoreTests.m

```diff
@@ -88,7 +88,6 @@ - (void) testCompileOptions
     XCTAssertTrue([options containsObject:@"ENABLE_FTS4"]);
     XCTAssertTrue([options containsObject:@"ENABLE_FTS3_PARENTHESIS"]);
     XCTAssertTrue([options containsObject:@"ENABLE_FTS5"]);
-    XCTAssertTrue([options containsObject:@"ENABLE_JSON1"]);
```

Comment:
Flag no longer returned on both iOS and Android / json1 is now always "baked in".